sprocs/sync_questions.sql

```diff
@@ -163,7 +163,8 @@ BEGIN
         workspace_enable_networking = (src.data->>'workspace_enable_networking')::boolean,
         workspace_environment = (src.data->>'workspace_environment')::jsonb,
         sync_errors = NULL,
-        sync_warnings = src.warnings
+        sync_warnings = src.warnings,
+        cross_origin_isolated = (src.data->>'cross_origin_isolated')::boolean
```

Comment:
This should come before `sync_errors` (it doesn't really matter, just convention within syncing code to have errors/warnings last).